src/Kuzzle.js

```diff
@@ -1,5 +1,6 @@
 var
   uuid = require('uuid'),
+  EventEmitter = require('./eventEmitter'),
```

Comment:
This is totally bad, we should not override build-in objects (Defined in nodejs context or browser one)